appveyor.yml

```diff
@@ -0,0 +1,49 @@
+version: 1.0.0_b{build}-{branch}
+
+os: Visual Studio 2015
+
+environment:
+  TERM: dumb
+  COVERALLS_REPO_TOKEN:
+    secure: 5nuCg+faxFPeppoNNcSwVobswAVFUf8ut83vw8CX/4W2y0kZkGmwEfCUxSQWiQDU
+  CI_NAME: appveyor
+  CI_JOB_ID: $(APPVEYOR_JOB_ID)
+  GCOV_PREFIX: $(APPVEYOR_BUILD_FOLDER)
+  matrix:
+  - COMPILER: msvc
+    GENERATOR: "Visual Studio 14 2015"
+    CFLAGS: /MP
+    CXXFLAGS: /MP
+  - COMPILER: mingw
+    GENERATOR: "MSYS Makefiles"
+
+platform:
+  - x86
+  - x64
+
+configuration:
+  - Debug
+  - Release
```

Comment:
Removed the release build. And added the artifacts rule which zips the build directory for each job (and downloadable from API and UI, e.g. https://ci.appveyor.com/project/am11/boxfort/build/1.0.0_b18-build/vcr/job/57iy8w6cmr6oo174/artifacts)